src/components/form/form/Terms.tsx

```diff
@@ -1,6 +1,12 @@
 import Checkbox from "@/components/Checkbox";
 
-const Terms = ({ options, toggle, onClick }) => {
+type terms = {
+  options: string[];
+  toggle: boolean;
+  onClick: () => {};
```

Comment:
() => void

needs to be void since it doesnt return anything

Reply:
checkout the other files for the format, it should be props and typed directly, dont need the react.fc